src/main/java/com/example/bookstoreapp/entities/MediaEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.example.bookstoreapp.entities;
+
+import com.example.bookstoreapp.models.MediaType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Table(name = "media")
+@Entity
+@NoArgsConstructor
+public class MediaEntity {
+    @Id
+   private Long id;
+  private   MediaType mediaType; // IMG, VIDEO
```

Comment:
Flyway script says the database column is tiny int but here it is declared as media type and this is inconsistent.